cmd/world/cardinal/stop.go

```diff
@@ -38,7 +37,8 @@ This will stop the following Docker services:
 		}
 		defer dockerClient.Close()
 
-		err = dockerClient.Stop(cmd.Context(), service.Nakama, service.Cardinal, service.NakamaDB, service.Redis)
+		services := getStartedServices(cfg)
+		err = dockerClient.Stop(cmd.Context(), services...)
```

Comment:
I think you can just put all service here like in the `purge` command. so you don't need to use `getStartedService` func

Reply:
a downside of this is that the output of `stop` won't match with the output of `start` which may cause some confusion, e.g. why is this container stopped even though it wasn't started?

I don't have a strong opinion here, so lmk if we want to proceed with removing `getStartedServices`

Reply:
update: we're stopping all containers in `world cardinal stop` because to make `start` and `stop` behavior match, we need to match the `--telemetry` flags too. it doesn't make sense to have a `--telemetry` flag for `world cardinal stop`.